users/user-models.js

```diff
@@ -1,3 +1,4 @@
+require('mongoose');
```

Comment:
I'm not sure how this works. I believe this should be `var mongoose = require('mongoose')`